Tools/WinMLRunner/src/CommandLineArgs.h

```diff
@@ -11,8 +11,8 @@ class CommandLineArgs
     bool IsUsingGPUMinPower() const { return m_useGPUMinPower; }
     bool UseBGR() const { return m_useBGR; }
     bool IsUsingGPUBoundInput() const { return m_useGPUBoundInput; }
-    bool IsIgnoreFirstRun() const { return m_ignoreFirstRun; }
```

Comment:
Ignore first run is being removed because we already separate first run from the iterations after the first iteration in this PR. Ignore first run originally was to be able to gather perf numbers without the first run polluting the numbers, but since with this change, we already separate it, there's no need for the user to specify this flag anymore.